server/client.go

```diff
@@ -4361,23 +4361,21 @@ func (c *client) flushAndClose(minimalFlush bool) {
 	}
 }
 
+var typeStringMap = map[int]string{
+	CLIENT: "Client",
```

Comment:
You need to run `go fmt` on this. You can see from the PR run that this file is not properly formatted.